vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -329,38 +339,92 @@ form {
   }
 }
 
-.select_wrapper {
-  max-width: 100%;
-  height: 28px;
-  background-color: #fff;
-  border: 1px solid $gray;
-  border-radius: $radius;
+// .select_wrapper {
+//   max-width: 100%;
+//   height: 28px;
+//   background-color: #fff;
+//   border: 1px solid $gray;
+//   border-radius: $radius;
+//   position: relative;
+//   vertical-align: middle;
+//   &:after {
+//     font-family: 'FontAwesome';
+//     content: '\f0d7';
+//     position: absolute;
+//     top: 3px;
+//     right: 10px;
+//     color: $darkerGray;
+//     pointer-events: none;
+//   }
+//   select {
+//     background: transparent;
+//     width: 100%;
+//     padding: 2px 28px 0px 14px;
+//     margin: 0;
+//     font-size: 12px;
+//     font-weight: 600;
+//     line-height: 18px;
+//     border: 1px solid transparent;
+//     border-radius: 0;
+//     height: 26px;
+//     -webkit-appearance: none;
+//     -moz-appearance: none;
+//     text-indent: 0.01px;
+//     text-overflow: '';
+//   }
+// }
+
+$styledSelectHeight: 2.25rem;
+
+.styled_select_wrapper {
   position: relative;
-  vertical-align: middle;
+  display: inline-block;
+  height: $styledSelectHeight;
+  width: 17rem; // around 240px
+  &.auto {
+    width: auto;
+  }
+  &.full {
+    width: 100%;
+  }
+  select {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    height: $styledSelectHeight;
+    opacity: 0;
+    z-index: 2;
+    &:focus + div {
+      border-color: $darkBlue;
+      &:after {
+        color: $darkBlue;
+      }
+    }
+    &.error + div {
+      border-color: $errorColor;
+    }
+  }
+}
+
+.styled_select {
+  height: $styledSelectHeight;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets